internal/host/formatdiskcmd.go

```diff
@@ -0,0 +1,58 @@
+package host
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"text/template"
+
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+type formatDiskCmd struct {
+	baseCmd
+}
+
+func NewFormatDiskCmd(log logrus.FieldLogger) *formatDiskCmd {
+	return &formatDiskCmd{
+		baseCmd: baseCmd{log: log},
+	}
+}
+
+func (d *formatDiskCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models.Step, error) {
+
+	var inventory models.Inventory
+	if err := json.Unmarshal([]byte(host.Inventory), &inventory); err != nil {
+		return nil, err
+	}
+	var steps []*models.Step
+	for _, disk := range inventory.Disks {
+		if disk.Bootable {
```

Comment:
I think the command should get the disks to format rather than having this logic itself.  Later we will probably allow users to choose.

Reply:
Not sure what do you mean, if the user will select the disks we will probably have the selection in the DB. 
Currently the command is called from get next step that have no logic inside it - it only gets the host id and call for host api, each step read relevant configuration from the DB and set the parameters according to it.
Where do you think this logic should be?

Reply:
My mistake, I read it wrong